webui/react/src/utils/experiment.ts

```diff
@@ -139,7 +139,11 @@ export const canExperimentContinueTrial = (
 ): boolean =>
   !experiment.archived &&
   !experiment.parentArchived &&
-  (!!trial || experiment?.numTrials === 1) &&
+  (!!trial ||
+    experiment?.numTrials === 1 ||
+    (experiment.state !== RunState.Completed &&
```

Comment:
`experiment.state !== RunState.Completed` is it needed? 
we have `terminalRunStates.has(experiment.state)`

Reply:
to add to this part, `terminalRunStates` include:
```
RunState.Canceled,
RunState.Completed,
RunState.Error,
RunState.DeleteFailed,
RunState.Deleted,
```
It seems like if it's `Completed` or `Deleted` it probably can not continue the trials. Might be worth defining a new curated list called `continuableRunStates`?

Reply:
Continue on multi trial experiment is only available for `Error` and `Cancel` state, so I ended up using the existing `erroredRunStates`